src/main/java/tk/valoeghese/manhattan/FunniChunkData.java

```diff
@@ -335,22 +339,24 @@ public static long key(int x, int z) {
 	}
 
 	public static Biome yeetImpl(int gx, int gz) {
-		Object game = FabricLoader.getInstance().getGameInstance();
-		MinecraftServer server = null;
+		if (GenBiome.server == null) {
```

Comment:
the client integrated server changes on world load, so you need to set GenBiome.server accordingly.
Your code here doesn’t do this, it will use the old object >:(